EIPS/eip-X-ASIC-resistant-block-mining.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: <to be assigned>
+title: Poisoning the Well
+author: David Stanfill <david@airsquirrels.com> 
+discussions-to: <email address>
+status: Draft
+type: Standards Track
+category Core
+created: 2018-04-03
+---
+
+
+## Simple Summary
+
+This EIP attempts to break ASIC miners specialized for the current ethash
+mining algorithm.
+
+
+## Abstract
+
+There are companies who currently have dedicated hardware based ethereum miners in
+production, and probabalistically actively mining.  This EIP aims to "Poison
+the well" by modifying the block mining algorithm in a manner that
+probabalistically *"breaks"* these miners if they are in-fact built on ASICs.
+
+
+## Motivation
+
+ASIC based miners will have lower operational costs than GPU based miners which
+will result in GPU based mining quickly becoming unprofitable.  Given that
+production of ASIC based miners has a high barrier to entry, this will cause a
+trend towards centralization of mining power.
+
+This trend towards centralization has a negative effect on network security,
+putting significant control of the network in the hands of only a few entities.
+
+Furthermore, Ethereum was initially designed as an ASIC resistant algorithm and 
+the community has voiced strong support for making a definitive stand on our position
+regarding dedicated mining hardware development to discourage future R&D investments.
```

Comment:
It is incorrect to interpret brigading on GitHub issues as "strong support".  Ethereum currently doesn't have a mechanism for determining actual support of an issue outside of the forking process.  Recommend removing the claims of "strong support" as they are not verifiable and may not actually be correct.

Reply:
Agree, proposing "Ethash remains ASIC resistant, however ASIC manufacturer technology is advancing  and ethash may require further changes in order to remain resistant to unforeseen
design techniques. This EIP seeks explicitly to buy time during which newly developed
ASIC technology will face a barrier while more long term mechanisms to ensure 
continued ASIC resistance can be explored."